build/scripts/build-server-rpm

```diff
@@ -114,6 +114,7 @@ build() {
 
             /usr/bin/createrepo_c --update /home/builder/rpm/RPMS
 
+            sudo sed -i -e 's/^#baseurl=/baseurl=/' -e 's/^metalink=/#metalink=/' /etc/yum.repos.d/epel.repo
```

Comment:
@matejkubinec FYI: I'll remove this as it didn't yield the result I was looking to.

Reply:
This needs to be then merged also to the main branch.

Reply:
Sure, go ahead!